test/Domain.Tests/UnitTests/Entities/HouseMemberExpenseTests.cs

```diff
@@ -1,6 +1,5 @@
 ﻿using BudgetBadgerWebApi.Domain.Entities;
 using Domain.Tests.TestData.Entities.HouseMemberExpense;
-using FluentAssertions;
 
 namespace Domain.Tests.UnitTests.Entities
```

Comment:
miss spell in test ContributionInPercentage_Should_Throw_ArgumentOutOfRangeException_For_Value_Otside_0_To_1_Range ->
ContributionInPercentage_Should_Throw_ArgumentOutOfRangeException_For_Value_Outside_0_To_1_Range

Reply:
Fixed